frontend/nginx.conf

```diff
@@ -18,8 +18,8 @@ server {
         proxy_pass         http://docker-app;
         proxy_redirect     off;
         proxy_set_header   Host $host;
-        proxy_set_header   X-Real-IP $remote_addr;
-        proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header   X-Forwarded-Host $server_name;
+        #proxy_set_header   X-Real-IP $remote_addr;
```

Comment:
if planning on using in production these should be uncommented. Proxy should set these for logging and tracking purposes.